src/libstd/io/stdio.rs

```diff
@@ -369,3 +369,10 @@ pub fn set_panic(sink: Box<Write + Send>) -> Option<Box<Write + Send>> {
         Some(s)
     })
 }
+
+#[doc(hidden)]
+pub fn _print(args: fmt::Arguments) {
+    if let Err(e) = stdout().write_fmt(args) {
+        panic!("failed printing to stdout: {:?}", e);
```

Comment:
Could this use `{}` instead of `{:?}`?

Also, could this be explicitly marked as `#[unstable]`?